service/src/main/java/uk/gov/hmcts/reform/fpl/service/SendDocumentService.java

```diff
@@ -73,6 +79,55 @@ public void sendDocuments(CaseData caseData, List<DocumentReference> documentToB
         return recipients;
     }
 
+    public List<Recipient> getSelectedOtherRecipients(CaseData caseData, List<Element<Other>> othersSelected) {
+        List<Recipient> recipients = new ArrayList<>();
```

Comment:
maybe that can be a bit more streamline with map(). recipients.addAll(new ArrayList<>((otherRepresentatives))) is a bit confusing to follow